docs/modules/misc/pages/integrations/quarkus.adoc

```diff
@@ -13,6 +13,17 @@ mvn quarkus:add-extension -Dextensions="one.microstream:microstream-quarkus-exte
 
 The extension requires Quarkus 2.11.1 as a minimal version.
 
+The integration requires Spring Boot 2.x (with version 2.1.0.RELEASE being the minimal supported one).
```

Comment:
Should be Quarkus instead of Spring Boot I guess

Reply:
Copy and paste error, fixed. Thx.